packages/js/src/preferences/types.ts

```diff
@@ -9,15 +9,14 @@ export type ListPreferencesArgs = {
   tags?: string[];
 };
 
-export type UpdatePreferencesArgs = {
```

Comment:
removed this type as it was not aligned with the general approach we follow everywhere